### DISCUSSION TITLE
winget export --winget-only

### DISCUSSION THREAD

**Discussion by exxmen** [microsoft/winget-cli#2653] [Ideas]
Is there a plan to add support to only export apps that were installed using winget? Maybe something like the "winget export --winget-only"

**Comment by denelon** [comment]
Not currently. WinGet can detect packages installed in the upgrade and list scenarios, so the initial design is to export everything that has a corresponding manifest available. A user might have installed something manually, and then during a `winget upgrade -all` the package was upgraded by WinGet. We'd need ways to reason about these things. There are some feature asks around "exclude" and some other scenarios that might also interplay with export.

**Comment by denelon** [comment]
Not currently. WinGet can detect packages installed in the upgrade and list scenarios, so the initial design is to export everything that has a corresponding manifest available. A user might have installed something manually, and then during a `winget upgrade -all` the package was upgraded by WinGet. We'd need ways to reason about these things. There are some feature asks around "exclude" and some other scenarios that might also interplay with export.
